ArduPlane/Attitude.cpp

```diff
@@ -392,6 +392,13 @@ void Plane::stabilize()
     }
 #endif
 
+#if AP_PLANE_GLIDER_PULLUP_ENABLED
+    if (pullup.in_pullup()) {
+        pullup.stabilize_pullup(get_speed_scaler());
+        return;
```

Comment:
really not a fan of this early return. I was trying to get everything to go through its respective `plane.control_mode->run()` function. Could this be in auto's run function?

Also note that because of the check on the next line we will reset the attitude controllers after the pull-up.

Reply:
yep, I've moved it to mode_auto in run()